### PR TITLE
hopefully fixed #7140 (the resize problem on mac)

### LINKED ISSUE #7140: PlatformImpl.BeginResizeDrag() does not work on Mac
The full code for the sample is located under https://github.com/npolyak/NP.Avalonia.Demos/tree/main/NP.Demos.VisualSamples/NP.Demos.CustomWindowResizeBug.
I created a very simple window without Chrome (I set SystemDecorations property to "None"). 
Create a red border on the right hand side for resizing. 

    <Window xmlns="https://github.com/avaloniaui"
            xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
	        xmlns:np="https://np.com/visuals"
            x:Class="NP.Demos.CustomWindowResizeBug.MainWindow"
            SystemDecorations="None"
            Background="LightGray"
            Width="400"
            Height="300">
	    <VisualLayerManager>
		    <Grid>
			    <Border Name="Right" 
			            Background="Red"  
			            Width="20"
			            Cursor="SizeWestEast"
			            HorizontalAlignment="Right"
			            VerticalAlignment="Stretch"/>
		    </Grid>
	    </VisualLayerManager>
    </Window>

Then, following "Decorated Window" example from the Catalogue, I found the border in the code behind and 
added a handler to its PointerPressedEvent:

    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();

            ...

            Border b = this.FindControl<Border>("Right");

            b.PointerPressed += B_PointerPressed;
        }

        private void B_PointerPressed(object? sender, PointerPressedEventArgs e)
        {
            PlatformImpl?.BeginResizeDrag(WindowEdge.East, e);
        }

       ...
    }

Window resize after pressing on the red border and dragging works fine on Windows 10 and linux, but does not work on Mac (at least it does not work on Mac Catalina)

### PR DESCRIPTION
**Pull request by npolyak**
Need to test that custom window (window with decorations) is resizable on mac. Hopefully fixes #7140

### CODE REVIEW

**Comment by robloo** [comment]
If you put 'fixes #7140' or 'closes #7140' GitHub will automatically link the issue to this PR and merging the PR will automatically close the issue. It just helps everyone keep track of things.

File: src/Avalonia.Visuals/Media/PixelPoint.cs [hunk 1]
```diff
@@ -252,4 +252,12 @@ public override string ToString()
             return string.Format(CultureInfo.InvariantCulture, "{0}, {1}", X, Y);
         }
     }
+
+    public static class PixelPointHelper
```
Comment (grokys) [inline src/Avalonia.Visuals/Media/PixelPoint.cs:256]: There's already the static `PixelPoint.FromPoint` method which does this.

Comment (npolyak) [reply src/Avalonia.Visuals/Media/PixelPoint.cs]: I just removed the method and switched the code to using PixelPoint.FromPoint(...) method instead.

Someone needs to test the change on Mac - I am temporarily out of Mac. I tested it on Windows with USE_MANAGED_DRAG set to true.

**Comment by npolyak** [comment]
Someone needs to test this change on Mac, since I am out of Mac temporarily. I tested the change on windows with USE_MANAGED_DRAG set to true and some one slight modifications.

**Comment by npolyak** [comment]
Friends, can someone please, do the review and the Mac test? I do not want this fix to go down the memory hole

**Comment by maxkatz6** [comment]
I was going to try it today, but found some problems with my mac, I don't usually use it. Will try it again later today.

**Comment by npolyak** [comment]
> I was going to try it today, but found some problems with my mac, I don't usually use it. Will try it again later today.

Dan tried it. it did not work, unfortunately. Thanks

**Comment by jmacato** [comment]
Closing due to inactivity and merge conflicts. Please ping me if you're gonna work on this again.

**Pull request closed without merging**